pkg/cloudprovider/providers/azure/azure_storage.go

```diff
@@ -20,57 +20,19 @@ import (
 	"fmt"
 
 	"github.com/Azure/azure-sdk-for-go/arm/storage"
-	"github.com/Azure/go-autorest/autorest/to"
 	"github.com/golang/glog"
 )
 
 const (
-	defaultStorageAccountType  = string(storage.StandardLRS)
-	fileShareAccountNamePrefix = "f"
+	defaultStorageAccountType      = string(storage.StandardLRS)
+	fileShareAccountNamePrefix     = "f"
+	sharedDiskAccountNamePrefix    = "ds"
```

Comment:
Those should be placed in the file where it is in use? 
azure_storage.go now contains file-service only code, maybe we need to merge it to azure_file.go someday.